packages/jellyfish-api-core/src/category/poolpair.ts

```diff
@@ -198,6 +217,20 @@ export interface PoolShareOptions {
   isMineOnly?: boolean
 }
 
+export interface PoolSwapMetadata {
+  from: string
+  tokenFrom: string
+  amountFrom: number
+  to: string
+  tokenTo: string
+  maxPrice?: number
+}
+
+export interface PoolSwapUTXO {
```

Comment:
Call this `UTXO` and rename everything in this file that was called `*UTXO` to use that single interface.

Similar to this https://github.com/DeFiCh/jellyfish/blob/015ec69bf41c73376ff1f659645e0095aceb7a42/packages/jellyfish-api-core/src/category/oracle.ts#L177-L180